src/ducks/apps/components/Discover.jsx

```diff
@@ -49,6 +49,8 @@ export class Discover extends Component {
     const title = <h2 className="sto-view-title">{t('discover.title')}</h2>
     return (
       <Content className="sto-discover">
+        {isFetching && <OneAppLoading />}
```

Comment:
I don't understand here, if I understand correctly, we should have the one app loading only when loading the application page, shouldn't we?
It seems like we will have the one app loading on Discover page but apps loading on MyApps page when reading the code

Reply:
That's where you are on an app on the store and press F5

Reply:
The way the fetching is working on cozy-store cannot allow us to distinguish between "are we fetching all the app or only one". However here, if we are fetching all the apps we are fetching the current one as well, so it looks ok to me.